src/Components/Header/index.jsx

```diff
@@ -3,7 +3,7 @@ import "./index.scss"
 
 export const Header = () => {
     return (
-        <header className='header container'>
```

Comment:
add class - container
in this className - add 
max-width: 1200px;
width: 100%;
padding: 10px;

Reply:
add container in <header className='header container'>